src/Symfony/Component/Routing/Generator/CompiledUrlGenerator.php

```diff
@@ -21,18 +21,33 @@
 class CompiledUrlGenerator extends UrlGenerator
 {
     private $compiledRoutes = [];
+    private $compiledAliases = [];
     private $defaultLocale;
 
     public function __construct(array $compiledRoutes, RequestContext $context, LoggerInterface $logger = null, string $defaultLocale = null)
     {
-        $this->compiledRoutes = $compiledRoutes;
+        // TODO: To be removed in Symfony 6.0
+        if (!isset($compiledRoutes['routes'])) {
+            trigger_deprecation('symfony/routing', '5.3', 'No idea.');
```

Comment:
Do you have any idea of a good deprecation message here ?

Reply:
`Providing a list of routes at the root level of the "$compiledRoutes" constructor argument of "__CLASS__" is deprecated, provide them in a "routes" subkey instead.`
My 2 cents..